src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -152,11 +181,39 @@ public final void refreshItems() {
 
     }
 
+    private int getClosedIssueSize() {
+        int count = 0;
+        for (GuiElement element: getElementsList()){
+            if (!element.getIssue().isOpen()){
+                count++;
+            }
+        }
+        return count;
+    }
+
     /**
      * This function updates the information in the panel footer.
      */
     private void updateFooter() {
-        totalLabel.setText(String.format("Total: %d ", issueCount));
+
+        if (issueCount == 0){
+            totalIssueText.setText(String.valueOf(0));
+            hideFooterPanelStatsDetails(true);
+        } else {
+            openIssueText.setText(String.valueOf(openIssueCount));
+            closedIssueText.setText(String.valueOf(closedIssueCount));
+            totalIssueText.setText(String.valueOf(issueCount));
+            hideFooterPanelStatsDetails(false);
+        }
+
+    }
+
+    private void hideFooterPanelStatsDetails(boolean hide) {
+        openIssueText.setVisible(!hide);
```

Comment:
Not sure if it's a good idea to have a `hideFooter...` method and pass in a boolean to either `hide` or `show`. @damithc